### PR TITLE
Update Other.json

### PR DESCRIPTION
**Pull request by Smashfanful**
Another Italian translation

### CODE REVIEW

File: android/assets/jsons/Translations/Other.json [hunk 1]
```diff
@@ -2673,9 +2674,6 @@
 		Simplified_Chinese:"开放的边界促进了彼此的了解，让我们的人民心心相通！"
 	}
 	
-	"Your so-called 'friendship' is worth nothing.":{ // When we have a decleration of friendship to someone and we declare war on them
```
Comment (yairm210) [inline android/assets/jsons/Translations/Other.json]: Why did you delete this?

Comment (Smashfanful) [reply android/assets/jsons/Translations/Other.json]: I don't remember I deleted this.

Comment (Smashfanful) [reply android/assets/jsons/Translations/Other.json]: Now I have corrected the issues here.
